optimizer_wrapper.rb

```diff
@@ -612,17 +612,22 @@ def self.apply_zones(vrp)
     }
   end
 
-  def self.clique_cluster(vrp, cluster_threshold, force_cluster)
+  def self.set_zip_condition(vrp, cluster_threshold)
     @zip_condition = cluster_threshold && vrp.matrices.any? && vrp.services.all?(&:activity) && vrp.rests.empty? &&
```

Comment:
While you are at it, can you remove the `@zip_condition` variable, please? It is used only locally in the `clique_cluster` function. 

Then we can name this function to `clique_cluster_candidate?` (we have this convention for dicho and split_solve) and its result can be assigned to a local variable `apply_clique_cluster` which can replace all `@zip_condition` inside `clique_cluster`.